sample-console-appender-client-cluster-consumer/src/main/resources/application.conf

```diff
@@ -4,7 +4,13 @@ MAPPER_CONFIG
 	{
 		"Person"=[
 		{
```

Comment:
It would be better if you add another mapping to another entity in addition to Person. The example showed how to use mapall. To show how to use nested entity you can add mapping to another entity.